WcaOnRails/db/migrate/20161212200704_add_maldives_to_countries.rb

```diff
@@ -0,0 +1,9 @@
+class AddMaldivesToCountries < ActiveRecord::Migration
+  def up
+    execute "INSERT INTO `Countries` (`id`, `name`, `continentId`, `iso2`) VALUES ('Maldives', 'Maldives', '_Asia', 'MV')"
```

Comment:
Just a small remark, since we have the table linked with the appropriate model you could do `Country.create(...)` I think.